client/jetpack-onboarding/main.jsx

```diff
@@ -53,7 +58,11 @@ export default connect(
 		const siteId = getUnconnectedSiteIdBySlug( state, siteSlug );
 		const settings = getJetpackOnboardingSettings( state, siteId );
 		const isBusiness = get( settings, 'siteType' ) === 'business';
-
+		const site = getUnconnectedSite( state, siteId );
+		const userId = site ? get( site, 'userEmail', null ) : '';
+		const hash = sha1();
+		hash.update( userId );
+		const userIdHashed = hash.digest( 'hex' );
```

Comment:
As we discussed in DM, seems like this will be a good candidate for a separate selector.

Reply:
I'll handle it in a separate PR, if that is ok.

Reply:
Separate PR is fine 👍